test/files/scalacheck/parallel-collections/pc.scala

```diff
@@ -1,71 +0,0 @@
-/*
```

Comment:
This gets removed in https://github.com/scala/scala/pull/5603 (and reinstated as a pure ScalaCheck test in https://github.com/scala/scala-parallel-collections/blob/master/scalacheck/src/test/scala/pc.scala)